.esbuild/plugins/qgds-plugin-clean-output-folders.js

```diff
@@ -1,30 +1,39 @@
 import fs from "fs";
 import log from "../helpers/logger.js";
 
+/**
+ * Clean the output folders /dist.
+ * Skip cleaning during the initial build or initial watch mode.
+ *
+ * @returns {Object} Object with a name and a setup function.
+ */
 export default function cleanFoldersPlugin() {
+  let isInitialBuild = true;
   return {
     name: "qgds-clean-output-folders",
     setup(build) {
       build.onStart(() => {
-        //Console feedback
-        log("cyan", `\u{1F4C2} Cleaning /dist folders\n\n`);
+        if (!isInitialBuild) {
```

Comment:
You set it to true, but never set it to false which disables this function logic.

Reply:
Thanks for the review, Will. I've fixed it on next commit.